### ISSUE TITLE
Installation  "error: failed to run custom build command for `gettext-sys v0.19.8`"

### ISSUE THREAD

**Issue by Ilphrin** [Plume-org/Plume#155]
Hi,

I just tried to run `cargo run` after installing all the prerequisites, the cargo build fail and shows me this error:

```
error: failed to run custom build command for `gettext-sys v0.19.8`
process didn't exit successfully: `/home/ilphrin/Perso/Site web/plume/Plume/target/debug/build/gettext-sys-3b747b165bc120c6/build-script-build` (exit code: 101)
--- stdout
cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_GETTEXT_SYSTEM
cargo:rerun-if-env-changed=GETTEXT_SYSTEM
cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_GETTEXT_DIR
cargo:rerun-if-env-changed=GETTEXT_DIR
cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_GETTEXT_BIN_DIR
cargo:rerun-if-env-changed=GETTEXT_BIN_DIR
cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_GETTEXT_LIB_DIR
cargo:rerun-if-env-changed=GETTEXT_LIB_DIR
cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_GETTEXT_INCLUDE_DIR
cargo:rerun-if-env-changed=GETTEXT_INCLUDE_DIR
OPT_LEVEL = Some("0")
TARGET = Some("x86_64-unknown-linux-gnu")
HOST = Some("x86_64-unknown-linux-gnu")
TARGET = Some("x86_64-unknown-linux-gnu")
TARGET = Some("x86_64-unknown-linux-gnu")
HOST = Some("x86_64-unknown-linux-gnu")
CC_x86_64-unknown-linux-gnu = None
CC_x86_64_unknown_linux_gnu = None
HOST_CC = None
CC = None
HOST = Some("x86_64-unknown-linux-gnu")
TARGET = Some("x86_64-unknown-linux-gnu")
HOST = Some("x86_64-unknown-linux-gnu")
CFLAGS_x86_64-unknown-linux-gnu = None
CFLAGS_x86_64_unknown_linux_gnu = None
HOST_CFLAGS = None
CFLAGS = None
DEBUG = Some("true")
running: "sh" "/home/ilphrin/Perso/Site web/plume/Plume/target/debug/build/gettext-sys-4b02346f75bd4451/out/gettext/configure" "--without-emacs" "--disable-java" "--disable-csharp" "--disable-c++" "--disable-shared" "--enable-static" "--enable-fast-install" "--with-included-gettext" "--with-included-glib" "--with-included-libcroco" "--with-included-libunistring" "--prefix=/home/ilphrin/Perso/Site web/plume/Plume/target/debug/build/gettext-sys-4b02346f75bd4451/out"
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... 
--- stderr
configure: error: unsafe srcdir value: '/home/ilphrin/Perso/Site web/plume/Plume/target/debug/build/gettext-sys-4b02346f75bd4451/out/gettext'
thread 'main' panicked at '
command did not execute successfully, got: exit code: 1

build script failed, must exit now', /home/ilphrin/.cargo/registry/src/github.com-1ecc6299db9ec823/gettext-sys-0.19.8/build.rs:251:5
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::print
             at libstd/sys_common/backtrace.rs:71
             at libstd/sys_common/backtrace.rs:59
   2: std::panicking::default_hook::{{closure}}
             at libstd/panicking.rs:211
   3: std::panicking::default_hook
             at libstd/panicking.rs:227
   4: std::panicking::rust_panic_with_hook
             at libstd/panicking.rs:463
   5: std::panicking::begin_panic_fmt
             at libstd/panicking.rs:350
   6: build_script_build::fail
             at ./build.rs:251
   7: build_script_build::run
             at ./build.rs:246
   8: build_script_build::main
             at ./build.rs:214
   9: std::rt::lang_start::{{closure}}
             at /checkout/src/libstd/rt.rs:74
  10: std::panicking::try::do_call
             at libstd/rt.rs:59
             at libstd/panicking.rs:310
  11: __rust_maybe_catch_panic
             at libpanic_unwind/lib.rs:105
  12: std::rt::lang_start_internal
             at libstd/panicking.rs:289
             at libstd/panic.rs:374
             at libstd/rt.rs:58
  13: std::rt::lang_start
             at /checkout/src/libstd/rt.rs:74
  14: main
  15: __libc_start_main
  16: _start

warning: build failed, waiting for other jobs to finish...
error: build failed
```

System: Pop_OS! 18.04 64 bits (Ubuntu)

Running with latest nightly rustc (with rustup) version

**Comment by Ilphrin** [comment]
Ok it seems like the crate can't compile if the working directory have a space (here, i have `/home/ilphrin/Perso/Site web/`)

It seems weird, but maybe it is a wanted behaviour from the crate, does anyone know if this is the case?

**Comment by audiodude** [comment]
Looks like it's running into:

https://www.gnu.org/software/autoconf/manual/autoconf-2.60/autoconf.html#Special-Chars-in-Variables

While building gettext maybe?

**Comment by audiodude** [comment]
Filed https://github.com/Koka/gettext-rs/issues/9

**Comment by Ilphrin** [comment]
Thanks for filling an issue on the appropriate repository!

**Comment by fdb-hiroshima** [comment]
Fixed by #327 (remove gettext-sys from dependecies)
